swagger_to_sdk_config.json

```diff
@@ -18,7 +18,7 @@
       ]
     },
     "azure_mgmt_authorization": {
-      "swagger": "arm-authorization/2015-07-01/swagger/authorization.json",
+      "swagger": "specification/authorization/resource-manager/readme.md",
```

Comment:
something seems incorrect here if i look at https://github.com/Azure/azure-sdk-for-python/blob/master/swagger_to_sdk_config.json

I do not see a key `swagger` looks like it should be `input-file`. Isn't it or I am missing something